packages/common/src/store/sdkUtils.ts

```diff
@@ -17,51 +10,3 @@ export function* getSDK() {
   const audiusSdk = yield* getContext<AudiusSDKContext>('audiusSdk')
   return yield* call(audiusSdk)
 }
-
-/** This effect is used to shadow a migration without affecting the return value.
- * It will run two effects in parallel to fetch the legacy and migrated responses,
- * compare the results, log the diff, and then return the legacy value. Errors thrown
- * by the effect for the migrated response will be caught to avoid bugs in the migrated
- * code from causing errors.
- * NOTE: Should always be called from within a saga.
- */
-export function* checkSDKMigration<T extends object>({
```

Comment:
Wait where did these migration utils go?

Reply:
I removed `checkSDKMigration` because we're no longer doing the shadowing plan. `compareSDKResponse` is still available and I've been using that during migration to iterate until the adapters are correctly producing matching output. But we don't need the fancy wrappers with sentry logging anymore.